operations/rates.md

```diff
@@ -63,23 +63,26 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 {
     "Rates": [
         {
-            "BaseRateId": null,
-            "BusinessSegmentId": null,
-            "GroupId": "c8b866b3-be2e-4a47-9486-034318e9f393",
             "Id": "ed4b660b-19d0-434b-9360-a4de2ea42eda",
+            "GroupId": "c8b866b3-be2e-4a47-9486-034318e9f393",
             "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
-            "IsActive": true,
+            "BaseRateId": null,
+            "BusinessSegmentId": null,
+            "IsActive": false,
             "IsEnabled": true,
             "IsPublic": true,
-            "Name": "Fully Flexible",
-            "ShortName": "FF",
-            "ExternalNames": {
-                "en-US": "Long Stay Flexible Rate"
-            },
-            "ExternalIdentifier": "D001"
+            "Type": "Public",
+            "Name": "Fully Flexible Rate",
+            "ShortName": "FLEXFLEX",
+            "ExternalNames": {},
+            "Description": {},
```

Comment:
We prefer to use real examples, rather than just blank. I guess it would be something like this:
```javascript
   "Description": {
      "en-US": "Fully flexible rate",
      "de-DE": "Völlig flexibler Tarif"
   },
```